install/kubernetes/helm/istio/values-istio-demo.yaml

```diff
@@ -1,10 +1,24 @@
 # This is used to generate istio.yaml for minimal, demo mode.
 # It is shipped with the release, used for bookinfo or quick installation of istio.
 # Includes components used in the demo, defaults to alpha3 rules.
+global:
+  proxy:
+    accessLogFile: "/dev/stdout"
+
+  disablePolicyChecks: false
```

Comment:
pls put an explanation for this.  i assume this means no mixer check regardless if mixer is installed.

Reply:
This is enabling policy check. The variable is documented in the values.yaml file.

Reply:
ok thanks!